src/BloomExe/ProjectContext.cs

```diff
@@ -231,6 +232,18 @@ protected void BuildSubContainerForThisProject(string projectSettingsPath, ICont
 //					}
 					});
 				});
+
+				//TODO: why doesn't this work? It gives 0 commands
+				//_commandAvailabilityPublisher = new CommandAvailabilityPublisher(_scope.Resolve<IEnumerable<ICommand>>());
```

Comment:
Is it possible that we have to first tell autofac how to resolve a request for IEnumerable<ICommand>, presumably with something that can enumerate the one-and-only instance of all the concrete subclasses?